moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1012,7 +1012,7 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
       std_msgs::msg::String event;
       event.data = "stop";
       trajectory_event_publisher_->publish(event);
-      rclcpp::spin_some(pnode_);
+      // rclcpp::spin_some(pnode_);
```

Comment:
I agree that there's no need to spin for publishing, please either remove this line or revert commenting this line

Reply:
Ups, the change was not supposed to be committed.
@JafarAbdi this issue with spinning was discussed [there](#1127). I'd like to make a change related to this but I don't know which strategy to choose.